### PR TITLE
Feature/contact book

### PR DESCRIPTION
**Pull request by Mir00r**

### CODE REVIEW

File: src/main/kotlin/com/example/coreweb/commons/ResourceUtil.kt [hunk 1]
```diff
@@ -8,11 +8,27 @@ import org.springframework.http.MediaType
 import org.springframework.http.ResponseEntity
 import java.io.File
 import java.io.FileInputStream
+import java.util.regex.Pattern
 
 class ResourceUtil private constructor() {
 
     companion object {
 
+        private const val phoneNumberRegexBD = "\\+?(88)?0?1[56789][0-9]{8}\\b" // this is for bangladeshi phone number
```
Comment (sayemoid) [inline src/main/kotlin/com/example/coreweb/commons/ResourceUtil.kt]: Why is this validators on resource util? Please move it to `com.example.common.utils.Validator`
As per I can say, phone and email validators already exists on that class. So you can just use them.

File: src/main/kotlin/com/example/coreweb/domains/contacts/models/entities/Contact.kt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    @Column(nullable = false)
+    lateinit var name: String
+
+    @Column(nullable = false, unique = true)
+    lateinit var phone: String
+
+    var email: String? = null
+
+    var address: String? = null
```
Comment (sayemoid) [inline src/main/kotlin/com/example/coreweb/domains/contacts/models/entities/Contact.kt]: Address Should not be string.

**Review by sayemoid — COMMENTED** [review]
@Mir00r can you please check and fix the issues?

File: src/main/kotlin/com/example/coreweb/domains/contacts/models/entities/Contact.kt (hunk 2 above)
Comment (Mir00r) [reply src/main/kotlin/com/example/coreweb/domains/contacts/models/entities/Contact.kt]: @sayemkcn vai then what will be the address field data type...?

Comment (sayemoid) [reply src/main/kotlin/com/example/coreweb/domains/contacts/models/entities/Contact.kt]: We need to create an address entity like `IntAddress` (International Address). It'll consist global address template. Let's sit tonight about it.

Comment (sayemoid) [reply src/main/kotlin/com/example/coreweb/domains/contacts/models/entities/Contact.kt]: Weight. I'll create an architecture to model how locations will work.

Comment (sayemoid) [reply src/main/kotlin/com/example/coreweb/domains/contacts/models/entities/Contact.kt]: @Mir00r please update the PR removing address field. We'll merge it, then after completing Location feature we'll add location there.

**Pull request merged**